pwiz_tools/Skyline/CommandArgs.cs

```diff
@@ -1923,15 +1972,22 @@ private bool ValidateArgs()
             return true;
         }
 
-        public static string WarnArgRequirementText(Argument usedArg, Argument requiredArg)
+        public static string WarnArgRequirementText(Argument usedArg, params Argument[] requiredArgs)
         {
-            return string.Format(Resources.CommandArgs_WarnArgRequirment_Warning__Use_of_the_argument__0__requires_the_argument__1_,
-                usedArg.ArgumentText, requiredArg.ArgumentText);
+            if (requiredArgs.Length == 1)
+                return string.Format(Resources.CommandArgs_WarnArgRequirment_Warning__Use_of_the_argument__0__requires_the_argument__1_,
+                    usedArg.ArgumentText, requiredArgs[0].ArgumentText);
+
+            var requiredArgsText = new string[requiredArgs.Length];
+            for (int i = 0; i < requiredArgs.Length; i++)
+                requiredArgsText[i] = requiredArgs[i].ArgumentText;
+            return string.Format(Resources.CommandArgs_WarnArgRequirementText_Use_of_the_argument__0__requires_one_of_the_following_arguments_,
```

Comment:
Seems like this could end up wrapping on an 80 character command prompt. Maybe use LineSeparate of the message and the arguments, which is guaranteed to be okay, no matter how many arguments are required.